src/js/views/renderCharacter.js

```diff
@@ -25,13 +25,22 @@ export const characterHTML = `
     </div>
 `;
 
-export default function renderCharacters() {
-  for (let i = 0; i < 4; i += 1) {
+export const createHeroListElement = () => {
+  const heroNode = document.createElement('li');
+  heroNode.classList.add('hero');
+  return heroNode;
+};
+
+export const insertCharacterHTML = (heroNode) => {
+  heroNode.insertAdjacentHTML('beforeend', characterHTML);
+};
+
+export default function renderCharacters(arr) {
+  arr.forEach((element) => {
```

Comment:
Nice Job on the `forEach` refactor

Reply:
I appreciate you redirecting me to MDN. I now see how the array works w/ forEach!

Reply:
There is one more thing you need to do that I did not recognize at first. In between line 38 and 39 you need to add the line `// eslint-disable-next-line no-unused-vars`. So it should look like this instead 
```
export default function renderCharacters(arr) {
  // eslint-disable-next-line no-unused-vars
  arr.forEach((element) => {

```
You need to do this to get the checks to pass. Usually it is not ok to work around ESLint like this. But in this case I am willing to say that yes it is ok to do this.

Reply:
OK I was wondering why I was getting the eslint error in the tests! I will fix it as you suggested, and look into it a bit more to understand it!